test/actions/ActionsProviderSpec.scala

```diff
@@ -558,6 +560,93 @@ class ActionsProviderSpec extends ControllerUnitTest
         SessionValues.VALID_TAX_YEARS -> validTaxYears))) shouldBe Redirect(UnauthorisedUserErrorController.show())
     }
 
+    "redirect to CisUser data finished urls" should {
```

Comment:
You already tested all of this in CisUserDataFinishedFilterActionSpec. You don't need to test the same thing twice. The only think you need to test here is that the new action is part of the actions composition. One happy case is enough.